cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -81,19 +81,10 @@ static const size_t egl_vendor_globs_len =
 // FIXME: this doesn't yet work with libGLX and libglvnd redirector
 // FIXME: this still doesn't work with the 361 driver
 static const char *nvidia_globs[] = {
-	"libEGL.so*",
 	"libEGL_nvidia.so*",
-	"libGL.so*",
```

Comment:
also should we modify the opengl interface to stop allowing access to some of these files from the /var/lib/snapd/hostfs/... directory too now that we won't be mounting them anymore?